style.css

```diff
@@ -1444,14 +1444,292 @@ body{
+        align-items: center;
+        justify-content: center; */
+        grid-template-columns: auto;
+    }
 
-@media screen and (min-width: 768px) and (max-width: 992px) { 
+    .about_left {
+        width: 300px;
+        height: 300px;
+    }
+
+    .about_left .text {
+        padding: 8px;
+    }
 
+    .about_left h1 {
+        transform: translateX(200px);
+        font-weight:300;
+        font-size: 18px;
+        margin:0;
+        text-shadow: 2px 2px 10px rgba(0,0,0,0.3);
+    }
+
+    .about_left h2 {
+        font-weight:600;
+        font-size: 14px;
+        font-style:italic;
+        color: #777777;
+        transform: translateX(200px);
+    }
+
+    .about_left .info_container {
+        display: grid;
+        grid-template-columns: auto auto;
+    }
+    .about_left p {
+        font-size: 10px;
+        margin: 0;
+        line-height: 35px;
+    }
+
+    .about-social-logo {
+        transform: scale(0.7, 0.7);
+        margin: 0;
+    }
+
+    .about_heading {
+        font-size: 18px;
+        letter-spacing: 1.5px;
+        margin-top: 30px;
+    }
+
+    .about_title {
+        font-size: 28px;
+    }
+
+    .about_msg span {
+        font-size: 12px;
+    }
+
+    .about_btn {
+        padding: 12px 20px;
+        text-align: center;
+    }
+
+    .about_sign_block {
+        display: grid;
+        grid-template-columns: auto;
+        margin-top: 20px;
+        margin-bottom: 20px;
+    }
+
+    .sign_img {
+        width: 60%;
+    }
+
+    .about_right a {
+        display: block;
+    }
+
+    .learning_heading {
+        font-size: 18px;
+        letter-spacing: 1.5px;
+    }
+
+    .learning_title {
+        font-size: 28px;
+    }
+
+    .learning_card_container {
+        grid-template-columns: auto;
+        margin: 0 auto;
+        margin-top: 10px;
+    }
+
+    .learning_card {
+        width: 100%;
+    }
+
+    .learning_description {
+        padding: 10px;
+    }
+
+    .learning_place {
+        font-size: 18px;
+    }
+
+    .learning_address {
+        font-size: 14px;
+        font-style: italic;
+        color: #777777;
+        font-weight: 400;
+        margin-bottom: 5px;
+    }
+
+    .learning_degree {
+        font-size: 12px;
+        font-weight: 600;
+    }
+
+    .overall_card_container {
+        width: 95%;
+        display: grid;
+        grid-template-columns: auto auto;
+        grid-gap: 20px;
+        margin: 0 auto;
+    }
+
+    .overall_card {
+        width: 100%;
+        min-width: 200px;
+        padding: 10px;
+    }
+
+    .overall_des_title {
+        font-size: 16px;
+    }
+
+    .overall_des {
+        padding: 5px;
+        font-size: 10px;
+    }
+
+    .skills_heading {
+        font-size: 16px;
+    }
+
+    .skills_li em {
+        font-size: 12px;
+        position: relative;
+        top: -30px;
+    }
+
+    /* PORTFOLIO */
+
+    #slide_down {
+        padding: 4% 4%;
+    }
+
+    .project__heading {
+        font-size: 22px !important;
```

Comment:
it's actually not a good practice to use !important unless absolutely necessary. May be you can look for other ways to achieve the same?

Reply:
I will try to do it i might overkill the code. Thank you very much for your suggestion. I will try :D